state.go

```diff
@@ -188,6 +188,12 @@ var (
 		},
 	}
 
+	inventoryUpdateState = &InventoryUpdateState{
+		BaseState{
+			id: MenderStateInventoryUpdate,
+		},
+	}
+
```

Comment:
Please update also the state machine schematic at the very top of the file. It would be nice to have it up to date.